### PR TITLE
Refine Bootstrap Styles and Semantic HTML

### PR DESCRIPTION
**Pull request by zachfedor**
I updated the views created by Alex with proper HTML5 elements and semantic tags. Then I matched the styles to the new html and tweaked it a bit. More tweaking is necessary, but this is a good launch point.

### CODE REVIEW

File: web/sass/custom.scss [hunk 1]
```diff
@@ -1,85 +1,120 @@
...
-    color: $text;
-    font-family: "Lato", sans-serif;
-    font-size: 16px;
+//
+// Variables
+// --------------------------------------------------
+$light-blue: #2d83ba;  // $body-bg
+$dark-blue:  #052E4C;  // $brand-primary
+$orange:     #F09740;  // $brand-success
+$gray:       #242A2F;  // $gray
```
Comment (brewcore) [inline web/sass/custom.scss]: We should be able to re-use the vars from bootstrap.  I'll take a look this weekend.

Comment (zachfedor) [inline web/sass/custom.scss]: not with the way we were using compass and linking the stylesheets. compass was watching all *.scss files, excluding those prefixed with an underscore, and compiling them to their own css. So styles.scss -> styles.css and custom.scss -> custom.css completely separate from each other, while _bootstrap-variables.scss was simply included in styles.scss. I rearranged things so custom.scss is now also an import in styles.scss, which means we only have to link the one and only compiled css file. i'll update the PR

**Comment by brewcore** [comment]
Is this ready to be merged now?

**Comment by zachfedor** [comment]
no, i need to update the PR

**Pull request merged**